src/test/scala/no/gdl/bookapi/repository/RepositoryTestHelpers.scala

```diff
@@ -39,6 +39,7 @@ trait RepositoryTestHelpers {
       language = language,
       datePublished = Some(LocalDate.now()),
       dateCreated = Some(LocalDate.now()),
+      dateArrived = dateArrived.getOrElse(LocalDate.now()),
```

Comment:
Just to be sure: The db should not set this field itself, with the `DEFAULT current_date`?

Reply:
This is just in a test trait, so this is only relevant for tests.
In production code, we read arrival date from the import script (so we have the option of overriding it). If no arrival-date is set, then the database will use `current_date` as the default